### PR TITLE
feat(module: avatar): add avatar group

### PR DESCRIPTION
**Pull request by ElderJames**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

https://ant.design/components/avatar-cn/#components-avatar-demo-group

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  add avatar group         |
| 🇨🇳 Chinese |  增加头像组样式          |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Review by TimChen44 — COMMENTED** [review]
1、鼠标在“+2”图标显示提示ku框时，鼠标移动到上一行绿色图标显示黑色提示框，然后移开后那个黑色提示框就不会消失
image

2、图标存在高低落差
image

File: components/avatar/Avatar.razor [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
+            <Icon Type="@Icon" />
+        }
+        @if (_hasSrc)
+        {
+            <img src="@Src" srcset="@SrcSet" alt="@Alt" @onerror="ImgError" />
+        }
+        @if (_hasText)
+        {
+            <span class="ant-avatar-string" @ref="TextEl" style="@_textStyles">
+                @if (!string.IsNullOrEmpty(this._text))
```
Comment (TimChen44) [inline components/avatar/Avatar.razor:18]: 我们大部分控件都是模板代码优先于属性值，这里我们是否可以统一成一样的规则？

**Comment by ElderJames** [comment]
> 1、鼠标在“+2”图标显示提示ku框时，鼠标移动到上一行绿色图标显示黑色提示框，然后移开后那个黑色提示框就不会消失
> image
> 
> 2、图标存在高低落差
> image

这个是tooltip的bug... 看看 @mutouzdl @Zhuangkh  能不能处理了

File: components/avatar/Avatar.razor (hunk 1 above)
Comment (ElderJames) [reply components/avatar/Avatar.razor]: 改了

**Review by TimChen44 — APPROVED** [review]

**Pull request merged**